docs/kyma-environment-broker/assets/keb-arch.drawio

```diff
@@ -0,0 +1 @@
+<mxfile host="app.diagrams.net" modified="2022-11-15T12:24:39.568Z" agent="5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36" etag="QT-9oxwwylNTxr21Lw2D" version="20.5.3" type="device"><diagram id="ZGsqVFqlMtBjAnkGU3Om" name="Page-1">5VpRd6I4FP41PnaOEER9rNZ2z7Y92zPuOTPdtwgBsxMJDaHV+fUbIAghqNhisesT5Ca5kHu/7ya5SQ9MV+s7BsPlI3UR6Zl9d90DNz3TNMZ2XzwSySaT2KaVCXyGXdmoEMzxbySFsp8fYxdFSkNOKeE4VIUODQLkcEUGGaNvajOPEvWrIfSRJpg7kOjSH9jly0w6MoeF/A+E/WX+ZcMeZzUrmDeWI4mW0KVvJRGY9cCUUcqzt9V6ikhivNwuWb/bHbXbH2Mo4E06zK3n0Z+PL/E/Dz//ZvYLmd39sK4GmZZXSGI5YPmzfJNbgNE4cFGixOiBydsSczQPoZPUvgmfC9mSr4ishgT7gXgnyBM/NfEwIVNKKEtVAReikecIecQZ/YVKNbYzQgtP1OijkgN9RYyjdUkkR3mH6ApxthFN8tq+tHgOOVl8K/xnDaRsWfIdyD0FJWb8rerCrOJFWvYIKw81oyJXoEwWKeNL6tMAklkhnRRm74tS0eaB0lAa+1/E+UZSBsacqq5Aa8x/lt6fE1XfBrJ0s5aa08JGFjwacKnQGO/zRkRj5qA9Q865CpmP+J52Em+JPfb6liECOX5VWdm6o4wcbAUfpgxBLiKQCDEkjjhi6c+8xCjitV59gAsRAmtJ4Qgriv5gkmAZixhzLStW2HUzp6MI/4aLVF/ij5DigKeDHEx6g5uKh4SdJiT53AQ6v/wUMDmlAhqgWv/tw6dGsW1Ylb+kRK466vW/9QEACvtyJDR2oFT+lAy90DxWlYKKAup5EeKa/7d/+H5IjLrkbsHX51JNPXeFi9mm1CkpPpfrim5pqTvOW+fF+f4ezt9vVjD9Ezn4/wvjRy0y3lKoabZCd0Odwq8M8Fl8H9esiGzCpSMU79svMc0rrqLURdeigWGG69QTeb1485PndRgSXJlKhLk97MdMGIsG+ZfEj2cfy/qdz+phuH/50FYIOhbibQco0DBAFRwwB7YC2By/76XB6XGur3TuZzrSDi33ozDbdHl4naCtleW7qXLfAPr6fVSzfB+davVuWJqpnhh9xZFgbJRTdsEKttpwldgpWETJ4z5eIBagyhryXBjdkNCGQueC3Q0IbX6IsmZDyhqtbyTSrteMwU2pgZyed85bg349eG93tLd3gL1p+4G66Rcv2R+3GirM3fivgbIK1AMBpIV4AUYVE/b1eLGNKeWAYZ0sYNiawb4jMdE7mJQWlV9uVzF895z+sRAAOgsBH0IB2A2CMyDNWCUNqJlkP5k0I81ejzCAfsqYFXXjlDtfjjRmR6SxviZp9KXWA/aQs3GEM81+Boju2WOCc2OPPuPIzMVUIIrRpP8TgYGewBAj5qp91Ny83HOVE/lS1DzHUecQ1WWNc1BHbCMqPhrWuKhuGwFOFuD0ZZT00fc44Hil++aok5dq6KgcvXieZzq1Ry+uvbDF7rWdaaWyPLXGDYlhn8zqh/M5+Zljnq4x9MzNdpfXJP8jbFif/zFK+Z1FVfXBnA8hOIx20ak5EAAYj29v64DQ5hncVYV9Rg0QrM8MkDnozgEH5uXgAOzYD3eHAz3v1RkOrAvCQWW/YdZs0j8XB/ps3BkOwOXgwK7MC1sHd4YDfZveGQ4Gl4MDqxIPrM7jgb7z7AwHw8vBgVndMHSOg7obcR3hwL4cHBjDyrww6hoHekZFM3G0hGGaNHR4YpMjDG32W9pmVe86Gtagkd0GJ7PbsMZuLd2fmKPATbLBf82FqLiE15APgXud3AcWJYfAKMKO6p2dCSkzT+9+4DpE76jT053AOHxaKlF7OO2btTuY9S3BqO7KbC774G0f7TSiehM3G7jsVSBUU2RZlTAyrCjKDKMpOvaY96rynTx8vfMYVhSLe+BZ8+I2PZj9Bw==</diagram></mxfile>
```

Comment:
the source file - this is for a case that someone will need a modification of the svg.